src/main/helm/helm-service/get-helm-release-resources/get-helm-release-resources.test.ts

```diff
@@ -66,150 +65,9 @@ describe("get helm release resources", () => {
 
       const actual = await actualPromise;
 
-      expect(actual).toEqual([]);
```

Comment:
These unit tests are completely removed instead of "updated" as proposed in commit! Making tested code untested code is a pretty big hit on quality, right? :)

Reply:
These unit tests were removed because the function they mock is not called anymore.

Reply:
Are you sure? I can see no unit test resulting to other than:

`response: []`

The interesting test would describe how to get some content therein?